cloudigrade/config/settings/base.py

```diff
@@ -30,7 +30,16 @@
 S3_DEFAULT_REGION = env('S3_DEFAULT_REGION', default='us-east-1')
 SQS_DEFAULT_REGION = env('SQS_DEFAULT_REGION', default='us-east-1')
 HOUNDIGRADE_AWS_AVAILABILITY_ZONE = env('HOUNDIGRADE_AWS_AVAILABILITY_ZONE',
-                                        default='us-east-1a')
+                                        default='us-east-1b')
+HOUNDIGRADE_AWS_AUTOSCALING_GROUP_NAME = env(
+    'HOUNDIGRADE_AWS_AUTOSCALING_GROUP_NAME',
+    default='EC2ContainerService-inspectigrade-test-bws-us-east-1b-'
+            'EcsInstanceAsg-JG9NX9WHX6NU'
+)
+HOUNDIGRADE_AWS_VOLUME_BATCH_SIZE = env.int(
+    'HOUNDIGRADE_AWS_VOLUME_BATCH_SIZE',
+    default=5
```

Comment:
Do we not want to do 32 like we originally talked about?

Reply:
Good catch. 👍  I accidentally overlooked that requirement.